### DISCUSSION TITLE
Clicking pseudo element in label does not toggle checkbox

### DISCUSSION THREAD

**Discussion by phuoccss** [adobe/react-spectrum#6093] [Q&A]
I'm making a checkbox with additional hint, associated via the aria-describedby attribute. I want to make it such that clicking the hint also toggle the checkbox, by creating a pseudo element for label and expand its bounding box. However, this does not work as expected using rac's Checkbox.  This is possible with the native checkbox.

Here's the link to the reproduction here https://codesandbox.io/p/devbox/old-butterfly-lm8wm4?file=%2Fsrc%2FApp.tsx%3A6%2C1.

Could you please advise on this? Thanks very much.

**Comment by snowystinger** [comment]
I can't see the codesandbox, can you ensure it's public?

**Reply by phuoccss** [reply]
my bad. I've just made it public.

**Reply by snowystinger** [reply]
No worries, I think you're running into this issue https://github.com/adobe/react-spectrum/issues/2506
